libs/model/src/utils/makeGetBalancesOptions.ts

```diff
@@ -43,13 +43,13 @@ export function makeGetBalancesOptions(
             });
             if (!existingOptions) {
               allOptions.push({
-                balanceSourceType: castedSource.source_type as any,
+                balanceSourceType: castedSource.source_type,
```

Comment:
the error is in the evmChainId field...not a big deal I think

Reply:
The error is only in `evmChainId` field because the `balanceSourceType` is not properly narrowed down to an EVM source. The type assertion works to remove the type error on `evmChainId`.

Reply:
I see.. fixed in #9322